common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -55,6 +56,7 @@ def _open_editor(self):
         self.course_wiki_page.open_editor()
         self.course_wiki_edit_page.wait_for_page()
 
+    @pytest.mark.skip(reason='This test fails when using the new coursehome MFE.')
```

Comment:
So what's going on with these? I'm wondering two things
1) Should these tests be updated to not fail and utilize the new MFE?
2) Why can't we use the override waffle flag here? (Even if we can, I think the first point supersedes this one)

Happy to help answer 1 if you're not sure, but wanted to ask you first